shared/src/commonMain/kotlin/world/respect/domain/opds/model/OpdsLink.kt

```diff
@@ -0,0 +1,26 @@
+package world.respect.domain.opds.model
+
+import kotlinx.serialization.Serializable
+
+/**
+ * Represents a Link Object in OPDS 2.0.
+ * The href field is required, all other fields are optional.
+ *
+ * For reference, see the schema: https://readium.org/webpub-manifest/schema/link.schema.json
+ */
+@Serializable
+data class OpdsLink(
```

Comment:
This is also missing properties from the spec as per https://readium.org/webpub-manifest/schema/link.schema.json

e.g. size, bitrate, language, alternate

As mentioned: please ensure that **all** properties are included 100% as per the spec.